CodingAChat/Scenes/Home/HomeCoordinator.swift

```diff
@@ -51,6 +51,7 @@ extension HomeCoordinator: HomeViewControllerDelegate {
         db.collection("channels").addSnapshotListener() { [weak self] (querySnapshot, err) in
             
             if let err = err {
+                UIAlertController.show(message: "Unable to load channels", from: self!.homeViewController)
```

Comment:
this self! maybe lead some crash. If you need strong self please add guard let self = self else { return }